java-squid/src/main/java/org/sonar/java/ast/parser/TreeFactory.java

```diff
@@ -265,6 +257,7 @@ public TypeArgumentListTreeImpl newTypeArgumentList(AstNode openBracketTokenAstN
     if (rests.isPresent()) {
       for (AstNode rest : rests.get()) {
         for (AstNode child : rest.getChildren()) {
+          // FIXME SONARJAVA-547 comma should be part of the ArgumentList as token
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)